### ISSUE TITLE
TypeError: unsupported operand type(s) for /: 'float' and 'Bicomplex'

### ISSUE THREAD

**Issue by bacalfa** [pbrod/numdifftools#39]
The following gives an error:

```
import numpy as np
import numdifftools as nd

g = lambda x: 1.0/(np.exp(x[0]) + np.cos(x[1]) + 10)

print(nd.Gradient(g, method="multicomplex")([1.0, 2.0]))
```

If I change the method to `"complex"`, it works fine. Is there a way to use the multicomplex method in this situation and other similar ones?

**Comment by pbrod** [comment]
I don't get this error:

    Python 2.7.14 |Anaconda custom (64-bit)| (default, Oct 15 2017, 03:34:40) [MSC v.1500 64 bit (AMD64)]

    In [10]: nd.Gradient(g, method="multicomplex")([1.0, 2.0]) 
    Out[10]: array([-0.01796112,  0.00600821])


What version of numdifftools and python are you using?

**Comment by bacalfa** [comment]
I'm on Windows 7 64-bit.

```
Python 3.6.5 :: Anaconda custom (64-bit)

numdifftools=0.9.20=py_0
```

**Comment by pbrod** [comment]
Fixed in commit [481caaf3e81] (https://github.com/pbrod/numdifftools/commit/481caaf3e81e01f048e4cb1b96f99bba919ce27a).

Closing.
